packages/models/cardstack/code-generator.js

```diff
@@ -37,6 +37,7 @@ define('@cardstack/models/generated/{{modelName}}', ['exports', '@cardstack/mode
      value: true
    });
    exports.default = _model.default.extend({
+     type: "{{modelName}}",
```

Comment:
This unlocks a bunch of other simplifications in places where we're using cs-model-type or model.constructor.name.